<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 107, in main
    result = linter.lint(
             ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 620, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 692, in lint_file
    messages = self.lint_recipe(
               ^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 748, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 247, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/check_build_help.py", line 555, in check_recipe
    o = int(path.split("/")[1])
        ^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: 'host'

ERROR conda.cli.main_run:execute(47): `conda run conda-lint /tmp/abs_1dtpqnvx8n/clone` failed. (See above for error)
</details>

--- a/recipe/build-core.bat
+++ b/recipe/build-core.bat
@@ -1,8 +1,11 @@
+bazel clean --expunge
+bazel shutdown
+
 cd python
 set SKIP_THIRDPARTY_INSTALL=1
 set IS_AUTOMATED_BUILD=1
 set "BAZEL_SH=%BUILD_PREFIX%\Library\usr\bin\bash.exe"
-"%PYTHON%" setup.py install
+"%PYTHON%" -m pip install . --no-deps --no-build-isolation
 rem remember the return code
 set RETCODE=%ERRORLEVEL%
 
@@ -11,7 +14,7 @@ rem different Python version) do not stumble on some after-effects.
 "%PYTHON%" setup.py clean --all
 
 rem Now shut down Bazel server, otherwise Windows would not allow moving a directory with it
-bazel "--output_user_root=%SRC_DIR%\..\bazel-root" "--output_base=%SRC_DIR%\..\b-o" clean
+bazel "--output_user_root=%SRC_DIR%\..\bazel-root" "--output_base=%SRC_DIR%\..\b-o" clean --expunge
 bazel "--output_user_root=%SRC_DIR%\..\bazel-root" "--output_base=%SRC_DIR%\..\b-o" shutdown
 rd /s /q "%SRC_DIR%\..\b-o" "%SRC_DIR%\..\bazel-root"
 rem Ignore "bazel shutdown" errors

--- a/recipe/build-core.sh
+++ b/recipe/build-core.sh
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -xe
 
+bazel clean --expunge
+bazel shutdown
+
 if [[ "${target_platform}" == osx-* ]]; then
   export LDFLAGS="${LDFLAGS} -lz -framework CoreFoundation -Xlinker -undefined -Xlinker dynamic_lookup"
 else
@@ -28,7 +31,7 @@ grep -lR ELF build/ | xargs chmod +w
 # now clean everything up so subsequent builds (for potentially
 # different Python version) do not stumble on some after-effects
 "${PYTHON}" setup.py clean --all
-bazel "--output_user_root=$SRC_DIR/../bazel-root" "--output_base=$SRC_DIR/../b-o" clean
+bazel "--output_user_root=$SRC_DIR/../bazel-root" "--output_base=$SRC_DIR/../b-o" clean --expunge
 bazel "--output_user_root=$SRC_DIR/../bazel-root" "--output_base=$SRC_DIR/../b-o" shutdown
 rm -rf "$SRC_DIR/../b-o" "$SRC_DIR/../bazel-root"
 # this is needed because on many build systems the cache is actually under /root.

--- a/recipe/build-core.sh
+++ b/recipe/build-core.sh
@@ -26,7 +26,7 @@ export SKIP_THIRDPARTY_INSTALL=1
 grep -lR ELF build/ | xargs chmod +w
 
 # now install the thing so conda could pick it up
-"${PYTHON}" setup.py install  --single-version-externally-managed --root=/
+${PYTHON} -m pip install . --no-deps --no-build-isolation
 
 # now clean everything up so subsequent builds (for potentially
 # different Python version) do not stumble on some after-effects

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -34,7 +34,7 @@ requirements:
     - m2-patch       # [win]
     - bazel ==4.2.2
     - curl
-    - cython ==0.29.28
+    - cython >=0.29.32
     - git
     - m2-make   # [win]
     - make      # [not win]

